content/customize-dock.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Dock (Move or Customize)
+description: >
+  How to move the Dock to different areas of the screen and customize other settings.
+keywords:
+  - Launcher
+  - Bottom
+  - Left
+  - Moving
+  - Dock
+  - Dash to Dock
+  - COSMIC Dock
+  - Ubuntu Dock
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+
+redirect_from:
+  - /movelauncher
+  - /articles/move-launcher
+---
+
+Pop!\_OS and Ubuntu both contain an application dock on the screen. The location of the dock, as well as other dock settings, can be adjusted in <u>Settings</u>.
+
+## Default Dock on Pop!\_OS
+
+Pop!\_OS includes a lightweight version of Dash to Dock.
```

Comment:
I already approved this PR, but I think it might be to change this to: 

>Pop!_OS's dock is positioned along the bottom edge of the screen by default.

Just because "lightweight" still doesn't make sense to me and the sentence feels like a placeholder. Up to you whether or not you want to implement it. Obvs. you'd want to make the Ubuntu one below match this one.

Reply:
Should be updated now so we can get another approval on this PR.